tests/Database/DatabaseEloquentBelongsToManySyncReturnValueTypeTest.php

```diff
@@ -90,7 +90,7 @@ public function testSyncReturnValueType()
         });
 
         $user->articles->each(function (BelongsToManySyncTestTestArticle $article) {
-            $this->assertEquals('0', $article->pivot->visible);
+            $this->assertSame('0', (string)  $article->pivot->visible);
```

Comment:
All of your `(string)` casting have two spaces after them. Can you reduce them to only one?

Reply:
done